apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/PromptResourceTest.java

```diff
@@ -315,4 +373,192 @@ Stream<Arguments> when__promptIsInvalid__thenReturnError() {
         }
     }
 
+    @Nested
+    @DisplayName("Find Prompt")
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    class FindPrompt {
+
+        @Test
+        @DisplayName("Success: should find prompt")
+        void shouldFindPrompt() {
+
+            String apiKey = UUID.randomUUID().toString();
+            String workspaceName = UUID.randomUUID().toString();
+            String workspaceId = UUID.randomUUID().toString();
+
+            mockTargetWorkspace(apiKey, workspaceName, workspaceId);
+
+            var prompt = factory.manufacturePojo(Prompt.class).toBuilder()
+                    .lastUpdatedBy(USER)
+                    .createdBy(USER)
+                    .build();
+
+            createPrompt(prompt, apiKey, workspaceName);
+
+            List<Prompt> expectedPrompts = List.of(prompt);
+
+            findPromptsAndAssertPage(expectedPrompts, apiKey, workspaceName, expectedPrompts.size(), 1, null);
+        }
+
+        @Test
+        @DisplayName("when search by name, then return prompt matching name")
```

Comment:
It's worth adding a test for the case of a prompt exists and you search for it with a string that doesn't match the name to see that it is not returned

Reply:
Done